test/components/Sources.spec.js

```diff
@@ -7,6 +7,12 @@ import sinon from 'sinon';
 
 
 describe('The component Sources', () => {
+
+  it('renders without exploding', () => {
```

Comment:
'renders without exploding'? this is not a very descriptive test spec. please be more descriptive in 6o lines or less